### DISCUSSION TITLE
Redirect cycle/loop when installing for the first time

### DISCUSSION THREAD

**Discussion by Sleepful** [firefly-iii/firefly-iii#9111] [Firefly III]
### Support guidelines

- [X] I've read the [support guidelines](https://github.com/firefly-iii/firefly-iii/blob/main/.github/support.md)

### I've found a bug and checked that ...

- [X] ... [the documentation](https://docs.firefly-iii.org/) does not mention anything about my problem
- [X] ... there are no open or closed issues that are related to my problem
- [X] ... it's [definitely a Firefly III issue, not me](https://github.com/firefly-iii/firefly-iii/blob/main/.github/its_you_not_me.md)

### Description

This was strange, the "loop" only occurred with nginx and HTTPS enabled.
if I tried to load my virtualHost (somedomain.tld) it would redirect me to `somedomain.tld/firefly-iii` through a 301, which would in turn redirect me to `somedomain.tld/firefly-iiifirefly-iii`, and so on, until the browser gave up.

I was lucky that I could go to `somedomain.tld/register` and create my account. After doing that the URL stopped acting weird.

Removing the HTTPS from nginx removed the issue, turning it back on would activate the issue again. 

### Debug information

I didn't find anything in the logs `journalctl --no-pager --since "2 hour ago" -u phpfpm-firefly-iii.service`

### Steps to reproduce

No idea, this was strange, it sounds similar to this other issue: https://github.com/firefly-iii/firefly-iii/issues/9004

### Additional info

Thanks for the software! I haven't had the time to really use it but it looks super polished

**Comment by JC5** [comment]
Weird indeed! Especially since Firefly III doesn't have any URLs that contain "firefly-iii". Could it be something in the nginx configuration?

**Reply by Sleepful** [reply]
Honestly I have no idea, but it is likely. It started working again after I made my first account. Must be the solar flares.... I didn't notice anything outside of the ordinary in the nginx config. 

FWIW I am using the preconfigured package from NixOS package manager. Not that it matters but yeah.

Just leaving the report over here in case someone else has the issue in the future, I don't think there's anything that can be done right now.
